content/03.results.md

```diff
@@ -32,8 +32,9 @@ Performance of models across four classification tasks. In each panel the loess
 ](./images/full_signal_combined.svg ){#fig:prediction_combined}
 
 ### There is predictive non-linear signal in transcriptomic data
-Our results to this point are consistent with a model where the predictive signal present in transcriptomic data is entirely linear.
+Our results to this point are consistent with a world where the predictive signal present in transcriptomic data is entirely linear.
 If that were the case, non-linear models like neural networks would fail to give any substantial advantage.
+However, based on past results we expect there to be relevant nonlinear biological signal [CITE clustermatch].
```

Comment:
Do you want to add the citation now? Looks like it just got preprinted: https://www.biorxiv.org/content/10.1101/2022.06.15.496326v1

Reply:
Definitely, thanks for the link!